src/Components/homepage/homepage.jsx

```diff
@@ -15,80 +20,226 @@ const useStyles = makeStyles(() => ({
+                    marginLeft: '2vw',
+                    fontSize: '4rem',
+                  }}
+                >
+                  <span style={{ color: 'blue' }}>R</span>esearch{' '}
+                  <span style={{ color: 'blue' }}>P</span>ortal
+                </p>
+              </div>
+              <div
+                className="row justify-content-center"
+                className={classes.btnSet}
               >
-                <span style={{ color: 'blue' }}>R</span>esearch{' '}
-                <span style={{ color: 'blue' }}>P</span>ortal
-              </p>
+                <Button
+                  component={Link}
+                  to="/projects"
+                  variant="outlined"
+                  className="col-2"
+                  className={classes.buttons}
+                  color="primary"
+                >
+                  Projects
+                </Button>
+                <Button
+                  component={Link}
+                  to="/login"
+                  variant="outlined"
+                  className="col-2"
+                  className={classes.buttons}
+                  color="primary"
+                >
+                  Login
+                </Button>
+              </div>
             </div>
-            <div
-              className="row justify-content-center"
-              className={classes.btnSet}
-            >
-              <Button
-                component={Link}
-                to="/projects"
-                variant="outlined"
-                className="col-2"
-                className={classes.buttons}
-                color="primary"
-              >
-                Projects
-              </Button>
-              <Button
-                component={Link}
-                to="/login"
-                variant="outlined"
-                className="col-2"
-                className={classes.buttons}
-                color="primary"
-              >
-                Login
-              </Button>
+            <div className="col text-center">
+              <img
+                src={introsvg}
+                alt=""
+                className="w-75 mr-4 d-flex "
+                style={{ marginTop: '10vh', minWidth: '250px' }}
+              />
             </div>
           </div>
-          <div className="col ">
-            <img
-              src={introsvg}
-              alt=""
-              className="w-75 mr-4 d-flex "
-              style={{ marginTop: '10vh' }}
-            />
+          <div className="row">
+            <div className="col-12" />
           </div>
         </div>
-        <div className="row">
-          <div className="col-12" />
+        <div className="container" />
+      </div>
+      {/* objectives */}
+      <div className="main-container">
+        <h1 className="text-center">Our Objectives</h1>
+        <div className="row ml-0 mr-0 text-center">
+          <div className="col-lg-3 col-md-3 col-sm-6 p-1 align-self-stretch">
```

Comment:
Don't use any bootstrap classes. Define these stylings inside the makeStyles hook of Material UI